libexec/mass-testing/symbolizer_wrapper

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
1. Rename all the scripts to use kebab-case, unless they are modules that need to be imported. Then we can use snake case.
2. Add a comment on the top of each script describing the purpose of the script.

Reply:
Ok for both